Gemfile

```diff
@@ -2,6 +2,8 @@ source "https://rubygems.org"
 
 gem "xcode-install", ">= 2.2.1" # needed for running xcode-install related tests
 
+gem "xcodeproj", ">= 1.5.3"
```

Comment:
This should already be a dependency of fastlane I think, check out `fastlane.gemspec` 👍